src/coreos-assembler

```diff
@@ -38,6 +38,9 @@ if [ -e /sys/fs/selinux/status ]; then
     fi
 fi
 
+# Start xinetd for Tang server
+sudo /usr/sbin/xinetd
```

Comment:
Eeek.  Let's only run this as part of the test.  Since this will only work in the qemu case, one approach is to use `ClusterSize: 0` and spawn a qemu instance directly and spawn tang there too.  

You can find an example of that over in [this PR](https://github.com/coreos/coreos-assembler/pull/1290/files#diff-04f8921039793708fe2af7c751d7b285R47).

Reply:
> Eeek. Let's only run this as part of the test. 

@cgwalters I'll move the execution/teardown to the test.

>Since this will only work in the qemu case, one approach is to use `ClusterSize: 0` and spawn a qemu instance directly and spawn tang there too.

Are you saying to launch two qemu instances--one to run tang and one as the system under test?

Reply:
No, still start it inside the cosa container, just do it as part of the test.  IOW it's only running when the test is running.  We don't want to run `xinetd` when we're doing our builds, etc.

Using two separate qemu VMs would be more isolated, but unfortunately qemu-unpriv networking makes that hard.